### ISSUE TITLE
Unable to show underscores

### ISSUE THREAD

**Issue by osban** [erikvullings/slimdown-js#3]
Slimdown is working very well, but I can't seem to be able to show actual underscore characters. Normally that should be possible by escaping with `\`.

**Comment by erikvullings** [comment]
I'm currently on holidays, but will have a look at it when back (Monday).

**Comment by erikvullings** [comment]
@osban I've just added a test for underscores, and underscores just pass through. See below test.

If I use `Hello\_world` in my test, my linter/prettifier automatically removes it, and if I use `Hello\\_world`, it remains in the output. So could you please specify what I/O behaviour you expect? Thanks

```ts
test('process underscores', (t) => {
  const expected = '<p>Hello_world</p>';
  const html = render('Hello_world');
  t.is(removeWhitespaces(html), removeWhitespaces(expected));
})
```

**Comment by osban** [comment]
@erikvullings it became apparent when I tried to use `<span class='material-icons'>mark_email_read</span>`, which didn't work as intended. You can also see it [here](https://flems.io/#0=N4IgtglgJlA2CmIBcAWAnAOjQZgDQgGd4EBjAF3imRAoLIzIJHwDMIEmkBtUAOwEMwiJDXh0GTfCQD2vCnOoQwAB2kAnMgAJgmtfF5R4azQF9NLNdLCaA5AVhKo0gO68AtACsCNgNwAdXgCZXjpNMChNAF5NAAMAgB5+TQALPRZIvxAWaWlMgD5ko3gAfX5i2jJ4gHp+PIC-P3KxMgbigJj-QN4wDD0DIwAKJxIAVyE5DAAjaSgAT1wwgZsoCAA3GwWesjURugG+wzUB8IBKM5OfZkJieHIIWU4QACYkAEYnkBMAXXwHXgBrTg8EACITUSBkVLsK4jNSwajJMhkZQEJBVKojXjKf4AcwwMjAVQhUNgAAFXhgKQA2IkQSFqdgYLxXMizZTCQgkBnKMifL4mIA) (using your live example).

**Comment by erikvullings** [comment]
@osban I've released v0.3.0, which allows you to use `\_`. 
Note that in order to pass `\_` in js code, you actually need to type `\\_`. [Live example](https://flems.io/#0=N4IgtglgJlA2CmIBcAWAnAOjQZgDQgGd4EBjAF3imRAoLIzIJHwDMIEmkBtUAOwEMwiJDXh0GTfCQD2vCnOoQwAB2kAnMgAJgmtfF5R4azQF9NLNdLCaA5AVhKo0gO68AtACsCNgNwAdXgCZXjpNMChNAF5NAAMAgB5+TQALPRZIvxAWaWlMgD5ko3g-Er8AfX5S8toyeIB6fjyAqrKaloCY-0DeMAw9AyMACicSAFchOQwAI2koAE9cMMGbKAgANxtF3rI1UbpB-sM1QfCASnPTn2ZCYnhyCFlOEBQkAA4QEwBdfAdeAGtODwQAIhNRIGRUuxrqM1LBqMkyGRlAQkHU6qNeMo-gBzDAyMB1cGQ2AAAQAjBgKQA2QkQCFqdgYLzXMhzZTCQgkBnKMgfT4mIA).
